app/javascript/packs/form-validation.js

```diff
@@ -62,9 +69,18 @@ function validateInput(input) {
  */
 export function initialize(form) {
   /** @type {HTMLInputElement[]} */
-  const fields = Array.from(form.querySelectorAll('.field'));
-  fields.forEach(validateInput);
-  form.addEventListener('submit', disableFormSubmit);
+  const fields = Array.from(form.querySelectorAll('.field,[required]'));
+  fields.forEach(addInputValidationListeners);
+  form.noValidate = true;
+  form.addEventListener('submit', (event) => {
+    fields.forEach(validateInput);
+    if (form.checkValidity()) {
+      disableFormSubmit(event);
+    } else {
+      event.preventDefault();
+      form.reportValidity();
```

Comment:
On some double-checking, I discovered this isn't supported in Internet Explorer 😞 

https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reportValidity#browser_compatibility

But actually,  it seems like the only changes to this file we might need are to include `[required]` in the selector above?

```diff
diff --git a/app/javascript/packs/form-validation.js b/app/javascript/packs/form-validation.js
index 5ba4eefef..34511e58c 100644
--- a/app/javascript/packs/form-validation.js
+++ b/app/javascript/packs/form-validation.js
@@ -62,7 +62,7 @@ function validateInput(input) {
  */
 export function initialize(form) {
   /** @type {HTMLInputElement[]} */
-  const fields = Array.from(form.querySelectorAll('.field'));
+  const fields = Array.from(form.querySelectorAll('.field,[required]'));
   fields.forEach(validateInput);
   form.addEventListener('submit', disableFormSubmit);
 }
```

I thought this might not be enough, but it seems to work in both Chrome and Internet Explorer at least (worth testing in other browsers as well).

I was thinking if that didn't work, the next thing would be to try to prevent the `invalid` event from triggering the normal form validation overlays.

Example:

```diff
diff --git a/app/javascript/packs/form-validation.js b/app/javascript/packs/form-validation.js
index 5ba4eefef..f8014a8b4 100644
--- a/app/javascript/packs/form-validation.js
+++ b/app/javascript/packs/form-validation.js
@@ -29,6 +29,14 @@ function checkInputValidity(event) {
 
   const { I18n } = /** @type {typeof window & LoginGovGlobal} */ (window).LoginGov;
 
+  if (
+    event.type === 'invalid' &&
+    !input.validity.valid &&
+    input.parentNode?.querySelector('.display-if-invalid')
+  ) {
+    event.preventDefault();
+  }
+
   if (input.validity.valueMissing) {
     input.setCustomValidity(I18n.t('simple_form.required.text'));
   } else if (input.validity.patternMismatch) {
@@ -62,7 +70,7 @@ function validateInput(input) {
  */
 export function initialize(form) {
   /** @type {HTMLInputElement[]} */
-  const fields = Array.from(form.querySelectorAll('.field'));
+  const fields = Array.from(form.querySelectorAll('.field,[required]'));
   fields.forEach(validateInput);
   form.addEventListener('submit', disableFormSubmit);
 }
```

Reply:
This looks like it works great. I tested it on my local...thanks!!!